src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -130,7 +134,7 @@
      * Define the RegExp to match the relief comment that suppresses
      * the warning about a fall through.
      */
-    private Pattern reliefPattern = Pattern.compile("fallthru|falls? ?through");
+    private Pattern reliefPattern = Pattern.compile("falls?[ -]?thr(u|ough)");
```

Comment:
This check and it's default values are part of google config. Please be sure that we want these texts for their config.

Reply:
https://google.github.io/styleguide/javaguide.html#s4.8.4-switch

>Any comment that communicates the idea of fall-through is sufficient (typically // fall through). 

So Google is ok with any comment.